harmony_changelog.md

```diff
@@ -8,6 +8,7 @@
 * Fixed alarm rings when deactivated during snooze
 * Fixed popup about file deletion showing in Relaxation app even if no file was deleted
 * Fixed displaying the menu after deep press on main screen
+* FIxed popup formatting for vertical home screen layout
```

Comment:
Changelog entry is not needed, as this is non-released bug :wink:

Reply:
done